src/OrbitGl/App.cpp

```diff
@@ -2042,6 +2043,19 @@ void OrbitApp::LoadPreset(const PresetFile& preset_file) {
   });
 }
 
+void OrbitApp::ShowPresetInExplorer(const PresetFile& preset) {
+  const auto file_exists = orbit_base::FileExists(preset.file_path());
+  if (file_exists.has_error()) {
+    ERROR("Unable to find preset file: %s", file_exists.error().message());
+    return;
+  }
+
+  std::string command = absl::StrFormat("explorer.exe /select,\"%s\"", preset.file_path().string());
+  if (system(command.c_str()) == 0) return;
+
+  ERROR("Unable to show preset file in explorer by running: %s", command);
```

Comment:
I think this should also be surfaced to the user. Maybe we don't wanna show the command in that case, but at least an error message that we were unable to open the file manager.

Reply:
Done! Thanks!